mappings/net/minecraft/block/impl/BlockStone.mapping

```diff
@@ -1,6 +1,24 @@
 CLASS none/aqc net/minecraft/block/impl/BlockStone
-	CLASS none/aqc$a
+	CLASS none/aqc$a StoneType
+		FIELD a STONE Lnone/aqc$a;
+		FIELD b GRANITE Lnone/aqc$a;
+		FIELD c SMOOTH_GRANITE Lnone/aqc$a;
+		FIELD d DIORITE Lnone/aqc$a;
+		FIELD e SMOOTH_DIORITE Lnone/aqc$a;
+		FIELD f ANDESITE Lnone/aqc$a;
+		FIELD g SMOOTH_ANDESITE Lnone/aqc$a;
+		FIELD h TYPES [Lnone/aqc$a;
+		FIELD i id I
+		FIELD j color Ljava/lang/String;
+		FIELD k name Ljava/lang/String;
+		FIELD l translationKey Lnone/ayp;
```

Comment:
Haven't looked at a lot of the localization mappings I believe this is correct though.